source/Score.js

```diff
@@ -0,0 +1,60 @@
+import Reference from "./Reference.js"
+
+
+export default class Score {
+    constructor(playerNumber) {
+        this.count = 0
+        this.playerNumber = (playerNumber === undefined) ? game.playerCount : playerNumber
+        this.domElement = document.createElement("div")
+        this.domElement.className += " player-score"
+        this.domElement.style.position = "absolute"
+        this.domElement.style.textAlign = "center"
+        var whiteText = document.createElement("div")
+        var colorText = document.createElement("div")
+        whiteText.className += " foreground-score"
+        colorText.id = "player-" + this.playerNumber + "-score"
```

Comment:
I've always been a fan of `.style.color = red` or whatever instead of putting them into their own CSS class. Keep it in the code, I say!